### PR TITLE
Jetpacks, Toy Hammer, Error, Trash, jackolanterns, Silver, Map Item resizes.

### PR DESCRIPTION
**Pull request by IProduceWidgets**
## About the PR

Resizes some things.
Fixes:
- Jackolanterns are now correctly graduated in size.
- Silver now appears correctly stacked in entity spawn panel.
- The handheld map has been resized due to its sprite (Normal).
- Jetpacks now have sizes that make any sense at all.

image
289331719-ad1b31f8-fc23-44ee-822a-9c603da630fc
image
(other items for size comparison only)
289233027-2a7ff3fa-1613-4f3b-9476-baab566bbaf5

Memes:
- Trash bags now appear to contain a rubbish bin storage grid. (I checked, the auto fill logic handles it fine, and it can be emptied into a trash chute on every corner.)
- Error Item Errors out (Its got a funny shape).
- Toy Hammer is cursed (funny shape).

image
289232796-428bded1-6689-4818-9ac1-8619602e4d6e
289313178-d0fa77d1-0690-48a8-b675-89d901a66491

## Why / Balance

Mostly fixes, the rest are memes.

In defense of the hammer: 
It makes the clown hammer hard to steal and keep away from the clown. Plus, the clown doesn't need all of their storage space anyway. Its also what we can point to any time someone wants a diagonal inv item.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Jetpacks have been correctly sized.

### CODE REVIEW

**Comment by IProduceWidgets** [comment]
Test fail is erroneous.

File: Resources/Prototypes/Entities/Objects/Specific/Janitorial/trashbag.yml [hunk 1]
```diff
@@ -11,7 +11,14 @@
...
     grid:
-    - 0,0,7,5
+    - 3,0,5,0
+    - 3,1,3,1
+    - 5,1,5,1
+    - 0,2,8,2
+    - 1,3,2,9
+    - 6,3,7,9
+    - 4,3,4,6
+    - 1,7,7,9
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Specific/Janitorial/trashbag.yml:21]: i don't really see the purpose in simultaneously buffing trashbags and making them awkward visually. the idea is fun but trashbags are too common of an item IMO to be getting this treatment. I'd rather just keep them as is as to not hamper their usability.

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 2]
```diff
@@ -946,6 +946,10 @@
       damage: 8
     - type: Item
       size: Small
+      shape:
+      - 1,0,1,0
+      - 1,1,2,1
+      - 0,2,0,2
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:952]: diagonal connections look really bad. i maybe try a shape like this?
```
XXX
.X.
```

or 

```
XXX
XXX
.X.
.X.
```

also, if the size is small, the total area needs to match that for consistency purposes. Personally, i don't really see the need to make it any bigger than 2x1. The joke isn't really that funny.

File: Resources/Prototypes/Entities/Objects/Fun/error.yml [hunk 3]
```diff
@@ -4,6 +4,12 @@
   name: error
   description: Hmmmm. Something went wrong.
   components:
+  - type: Item
+    shape: # funny shape
+    - 0,0,6,0
+    - 0,1,0,2
+    - 0,3,6,3
+    - 6,1,6,2
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Objects/Fun/error.yml:12]: due to the limits of the grid piece rendering, i would prefer if there wasn't  a gap in the center of the shape. It looks extremely buggy and will be covered up, functionally obscuring it. Maybe just do it like so:
```
XXXXXXX
X.XXX.X
X.XXX.X
XXXXXXX
```

again, this is another one where i'm not really sure what the purpose is. it's fine if it's just for gaffs but it has to look good. Then again, the shape is also really big so i don't even know if this is something we want.

**Review by EmoGarbage404 — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/Janitorial/trashbag.yml (hunk 1 above)
Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Objects/Specific/Janitorial/trashbag.yml:21]: Do people open trashbags regularly? When I played jani before making this I honestly thought the trash bags were disposable. It seems like the trashbag life cycle is click on ground to use auto fill -> click on disposal to empty with no reason i can think of to rummage around in it really.

File: Resources/Prototypes/Entities/Objects/Fun/error.yml (hunk 3 above)
Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Objects/Fun/error.yml:12]: It is obscured if other items are added to the bag, but it doesn't prevent function.
The error can be moved by clicking on the grid, and items placed in the middle can be interacted with as usual.

There's no reason to it, its just a funny error item.

File: Resources/Prototypes/Entities/Objects/Specific/Janitorial/trashbag.yml (hunk 1 above)
Comment (arimah) [reply Resources/Prototypes/Entities/Objects/Specific/Janitorial/trashbag.yml:21]: > Do people open trashbags regularly?

I sure do! Not all locations have disposal units right around the corner, and sometimes I wanna see how full it is before venturing into a trail of bullet casings. Granted, judging fullness was easier before gridinv, but even so. Plus, sometimes you wanna take things out that you didn't mean to hoover up.

This wonky interior shape really does feel needlessly gimmicky. It's literally just a bag, why does it need a silly shape?

**Comment by chromiumboy** [comment]
Hi @IProduceWidgets, are you coming back to this?

**Comment by IProduceWidgets** [comment]
I'm going to close this because It probably needs to be atomized again again again again since one or two things were not liked.

**Pull request closed without merging**